src/lib/translators/mongo.ts

```diff
@@ -2330,8 +2330,8 @@ export class Mongo36Translator extends BaseTranslator {
       return {
         $expr: {
           [operatorMapping[cond.operator]]: [
-            this.truncateColumnToDay(cond.column),
-            isRelativeDate(cond.value) ? this.translateRelativeDate(cond.value) : cond.value,
+            this.truncateDateToDay($$(cond.column)),
```

Comment:
Je comprends pas pourquoi on applique le truncate aux deux (colonne/value) ici ?

Reply:
Good point! I'm gonna add comments

The idea is that we can never be so sure of the time of the variables that are gonna be passed to weaverbird.
Imagine we have a variable "now" that means `2021-30-11T10:32`, and someone makes a query with the filter step:
`{ column: 'dateCol', operator: 'until', value: '<%= now %>' }`.
Then the query translated in mongo will be roughly: 
`{ $match: { $expr: $lte: [ '$dateCol',  '<%= now %>']}}`.
If we replace the variablein this and execute the query, we will have the wrong hours (dateCol <= 2021-30-11T10:32).
So we need mongo to remove the hours on both side to make sure we comapre only days.

Reply:
d8c8b76